recipes/jq/jq.inc

```diff
@@ -21,3 +21,6 @@ AUTO_PACKAGE_LIBS = "jq"
 AUTO_PACKAGE_LIBS_DEPENDS = "${DEPS}"
 AUTO_PACKAGE_LIBS_RDEPENDS = "${DEPS}"
 FILES_jq-libjq-dev += "${includedir}"
+
+RECIPE_FLAGS += "jq_disable_maintainer_mode"
+EXTRA_OECONF:>USE_jq_disable_maintainer_mode = " --disable-maintainer-mode"
```

Comment:
because I hadn't found the readme that actually explained what this was about. Yes, it makes sense to set unconditionally. Pushed.